server/routerlicious/packages/services-utils/src/auth.ts

```diff
@@ -33,6 +33,18 @@ import {
 } from "@fluidframework/server-services-telemetry";
 import { getBooleanFromConfig, getNumberFromConfig } from "./configUtils";
 
+interface IKeylessTokenClaims extends ITokenClaims {
+	/**
+	 * Identifies if the token is for Keyless Access or not.
+	 */
+	fluidRelayKeylessAccess?: boolean;
```

Comment:
If the client is supposed to be able to send this information, then this should be exposed/exported in a client-facing library (e.g. services-core)

Reply:
The client would not be able to send this claim to us. This will be a server generated claim